agent/browser/ie/wptbho/wpt.cc

```diff
@@ -23,9 +23,25 @@ static const TCHAR * REG_SHELL_FOLDERS =
     _T("Software\\Microsoft\\Windows\\CurrentVersion")
     _T("\\Explorer\\User Shell Folders");
 static const TCHAR * LOAD_EVENT_END_FUNCTION =
-    _T("var wptGetLoadEventEndTiming = (function(){")
-    _T("  return performance.timing['loadEventEnd'];")
-    _T("});");
+    _T("window.wptGetLoadEventEndTiming = function(){")
+    _T("  var t = performance.timing['loadEventEnd'];")
+    _T("  return (!t && window.__loadEventEndTiming ? window.loadEventEndTiming : t);")
+    _T("};");
+
+// Workaround to collect load event start/end when performance nav timing fails due to IE10 bug
+static const TCHAR * REGISTER_LOAD_EVENT_END_FUNCTION =
+    _T("window.__loadEventStartTiming = 0;")
+    _T("window.__loadEventEndTiming = 0;")
+    _T("(function(){")
+    _T("  window.addEventListener(\"load\", function () {")
+    _T("    setTimeout(function () { window.__loadEventEndTiming = Date.now(); }, 0);")
+    _T("  });")
+    _T("})();")
+    _T("window.wptMarkLoadEventStartTiming = function(){")

```

Comment:
Can we move this to a separate TCHAR *? It's confusing that the 'start' timing setup is in a constant named as 'end'